packages/dbus-broker/dbus-broker.spec

```diff
@@ -53,11 +53,11 @@ install -p -m 0644 %{S:3} %{buildroot}%{_cross_sysusersdir}/dbus.conf
 %{_cross_bindir}/dbus-broker-launch
 %dir %{_cross_datadir}/dbus-1
 %{_cross_datadir}/dbus-1/*
-%{_cross_journalcatalogdir}/dbus-broker.catalog
-%{_cross_journalcatalogdir}/dbus-broker-launch.catalog
+%{_prefix}/lib/systemd/catalog/dbus-broker.catalog
+%{_prefix}/lib/systemd/catalog/dbus-broker-launch.catalog
 %{_cross_sysusersdir}/dbus.conf
 %{_cross_unitdir}/dbus-broker.service
 %{_cross_unitdir}/dbus.socket
-%exclude %{_cross_userunitdir}/dbus-broker.service
+%exclude %{_prefix}/lib/systemd/user/dbus-broker.service
```

Comment:
The previous macros are correct; we need to make sure we install files under the sys-root path rather than directly under `/usr`.

Looking at dbus-broker's meson.build file, it uses pkg-config to find these paths from systemd.pc. I'd guess something changed in the systemd update, and I expect we'll have to fix it there.

In the `build/rpms` directory, you can use these commands to inspect the file:
```
rpm2cpio bottlerocket-aarch64-systemd-devel-247-1.aarch64.rpm |cpio -idmv
cat ./aarch64-bottlerocket-linux-gnu/sys-root/usr/lib/pkgconfig/systemd.pc
```

The current file has these lines:
```
prefix=/aarch64-bottlerocket-linux-gnu/sys-root/usr
root_prefix=/aarch64-bottlerocket-linux-gnu/sys-root/usr
rootprefix=${root_prefix}
```

If the prefix is just `/usr` now then that would explain this behavior.

Reply:
prefix is set to `/usr/` in v247. see https://github.com/systemd/systemd/commit/6e65df89c348242dbd10036abc7dd5e8181cf733 :facepalm:

Reply:
This has been reverted in in upstream main branch https://github.com/systemd/systemd/commit/60bce7c6d9606185114df1bdcd5ea100407688b8

Reply:
I appreciate upstream's point but it doesn't really apply to us: we ensure `/usr` points to the sys-root path at runtime, but for our own packaging we need to be careful to avoid dependencies on the host OS.

Either we should just do a straight revert patch of https://github.com/systemd/systemd/commit/6e65df89c348242dbd10036abc7dd5e8181cf733, or else add https://github.com/systemd/systemd/commit/60bce7c6d9606185114df1bdcd5ea100407688b8 and add a local patch to restore `prefix=@prefix@` (if needed, let's try it without).

Reply:
I'll go ahead and make a revert patch of https://github.com/systemd/systemd/commit/6e65df89c348242dbd10036abc7dd5e8181cf733.